compose.yaml

```diff
@@ -54,6 +54,7 @@ services:
     environment:
       - MONGO_DB_URI=mongodb://mongo:27017
       - MONGO_DB_NAME=${MONGO_DB_NAME}
+      - UI_URL=${UI_URL}
```

Comment:
Have we added this to the environment variables used by the deployment pipeline in Github?

Reply:
I don't see where this new variable is used, was it only required for creating the absolute uri which isn't used any more?

Reply:
yes, this was only used for the absolute path for local testing. deleted.